pkg/sql/logictest/testdata/logic_test/array

```diff
@@ -1333,11 +1333,16 @@ SELECT x, y FROM t ORDER BY (x, y)
 
 subtest array_indexes
 
-# Create indexes on arrays.
+# Indexes not yet supported for arrays, see #50659.
 statement ok
-DROP TABLE IF EXISTS t;
+DROP TABLE IF EXISTS t
+
+statement error column x is of type int\[\] and thus is not indexable
 CREATE TABLE t (x INT[] PRIMARY KEY)
 
+statement ok
+CREATE TABLE t (x INT[])
```

Comment:
shouldn't this fail now?

Reply:
Not sure what you mean. You can create a table with []int, just not with an index on it.

Reply:
woops, brain fart.